packages/apollo-server-koa/src/ApolloServer.ts

```diff
@@ -0,0 +1,178 @@
+import * as Koa from 'koa';
+import * as KoaRouter from 'koa-router';
+import * as corsMiddleware from '@koa/cors';
+import * as bodyParser from 'koa-bodyparser';
+import {
+  renderPlaygroundPage,
+  RenderPageOptions as PlaygroundRenderPageOptions,
+} from '@apollographql/graphql-playground-html';
+import { ApolloServerBase, formatApolloErrors } from 'apollo-server-core';
+import * as accepts from 'accepts';
+import * as typeis from 'type-is';
+
+import { graphqlKoa } from './koaApollo';
+
+import { processRequest as processFileUploads } from 'apollo-upload-server';
+
+export { GraphQLOptions, GraphQLExtension } from 'apollo-server-core';
+import { GraphQLOptions, FileUploadOptions } from 'apollo-server-core';
+
+export interface ServerRegistration {
+  // Note: You can also pass a connect.Server here. If we changed this field to
+  // `express.Application | connect.Server`, it would be very hard to get the
+  // app.use calls to typecheck even though they do work properly. Our
+  // assumption is that very few people use connect with TypeScript (and in fact
+  // we suspect the only connect users left writing GraphQL apps are Meteor
+  // users).
+  app: Koa;
+  path?: string;
+  cors?: corsMiddleware.Options | boolean;
+  bodyParserConfig?: Object | boolean;
```

Comment:
Let's infer the Options interface for now and leave a comment pointing at DefinitelyTyped/DefinitelyTyped#27047. Thank you so much for opening that PR @chentsulin!